librz/asm/arch/tms320/c55x_plus/hashtable.c

```diff
@@ -443,6 +443,7 @@ st32 get_hashfunc_10(st32 arg1, st32 arg2) {
 }
 
 st32 get_hashfunc_11(st32 arg1, st32 arg2) {
+#if 0
```

Comment:
? delete

Reply:
It was a reverse-engineered code (the whole instruction set is undocumented). I think it's better to convert it to the proper comment but leave it here.